perf.groovy

```diff
@@ -74,6 +74,14 @@ def static getOSGroup(def os) {
 					batchFile("py \"%WORKSPACE%\\Microsoft.BenchView.JSONFormat\\tools\\machinedata.py\"")
 					batchFile("set __TestIntermediateDir=int&&build.cmd ${configuration} ${architecture}")
 					batchFile("tests\\runtest.cmd ${configuration} ${architecture} GenerateLayoutOnly")
+
+                    if (arch == 'x86jit32')
+                    {
+                        // Download package and copy compatjit into Core_Root. We are using a set version until the packaging of x86 compatjit.dll works again
+                        batchFile("C:\\Tools\\nuget.exe install runtime.win7-${architecture}.Microsoft.NETCore.Jit -Source https://dotnet.myget.org/F/dotnet-core -OutputDirectory \"%WORKSPACE%\" -Prerelease -ExcludeVersion\n" +
```

Comment:
It looks like this grabs the latest version of this package. Is this what we want?  It seems to be the opposite of what the comment says.